testdata/go/memcached-operator/api/v1alpha1/memcached_webhook.go

```diff
@@ -1,5 +1,5 @@
 /*
-
+Copyright 2020.
```

Comment:
These changes in the scaffold is because of the bugfix fix `copyright` info on the license of the generated files when the owner is not informed.
      More info: https://github.com/kubernetes-sigs/kubebuilder/pull/1749